content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +85,142 @@ func TestAccXxx(*testing.T)
+
+For example, the following Makefile configuration:
+
+```make
+testacc:
+  TF_ACC=1 go test -v ./...
 ```
-$ make testacc
+
+Enables developers to use the following to run acceptance tests:
+
+```shell
+make testacc
 ```
 
-**It’s important to reiterate that acceptance tests create actual cloud resources**,
-possibly incurring expenses which are the responsibility of the user running
-the tests. Creating real infrastructure in
-tests verifies the described behavior of Terraform Plugins in real world use
-cases against the actual APIs, and verifies both local state and remote values
-match. Acceptance tests require a network connection and often require
-credentials to access an account for the given API.
+### GitHub Actions
+
+Create a [GitHub Actions workflow](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) file, such as `.github/workflows/test.yaml`, that:
+
+- Runs when pull requests are submitted (or on [other events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows) as appropriate)
+- Uses [`actions/checkout`](https://github.com/actions/checkout) to checkout the provider codebase
+- Uses [`actions/setup-go`](https://github.com/actions/setup-go) to install Go
+- Uses [`hashicorp/setup-terraform`](https://github.com/hashicorp/setup-terraform) to install Terraform CLI
+- Runs the `go test` command with the appropriate environment variables and flags
+
+For example:
+
+```yaml
+name: Terraform Provider Tests
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/test.yaml'
+      - '**.go'
+
+permissions:
+  # Permission for checking out code
+  contents: read
+
+jobs:
+  acceptance:
+    name: Acceptance Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: '1.1.*'
+          terraform_wrapper: false
+      - run: go test -v -cover ./...
+        env:
+          TF_ACC: '1'
+  unit:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+      - run: go test -v -cover ./...
+```
 
-~> Note: When developing or testing Terraform plugins, we highly recommend
-running acceptance tests with an account dedicated to testing. This ensures no
-infrastructure is created or destroyed in error during development or validation
-of any Provider Resources in any environment that cannot be completely and
-safely destroyed.
+More advanced configuration, such as running acceptance testing against multiple Terraform CLI versions, is possible via the [`matrix`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) strategy.
+
+For example, adjusting the job configuration:
```

Comment:
When we have an example I like to format it as such: 

The following example....[explain what it does]